Privacy-policy.md

```diff
@@ -0,0 +1,12 @@
+# Privacy Policy
+
+## Effective date: September 06, 2018
+[Codemancers](https://codemancers.com) operates the Interceptor browser extension. ([Chrome](https://chrome.google.com/webstore/detail/interceptor/enenfaicdcfgcnjmiigcjbmlbaoapnen) and [Firefox](https://addons.mozilla.org/en-US/firefox/addon/xhr-interceptor/)).
+
+Interceptor browser extension does not collect or share personal information of any kind. Interceptor does not share any data accepted from the users with third parties.
```

Comment:
Why the second sentence. we are not collecting anything. not even we are putting any third-party analytics.

Reply:
Yes, we dont collect any data from users. Its seems like a standard template for privacy policy. So lets merge this for now ?